ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4177,7 +4205,15 @@ MWVPM:
   - makeup_water_valve_percentage_command
   implements:
   - MONITORING
-
+  
+MWVVM:
+  guid: "0e4215df-3401-4cf0-bdfe-9e03d6c69a48"
+  description: "Make-up water valve voltage monitoring."
+  is_abstract: true
+  uses:
+  - makeup_water_valve_voltage_command
```

Comment:
please confirm that there is no sensor

Reply:
No sensor, see my comment for PHWVVSC

Reply:
Similar to SVVM, if there is not temperature control it would be prefereable to have a voltage sensor. I would recommend adding one and marking it as missing in the building config.

Reply:
Added voltage sensor for MWVVM